its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -177,14 +177,20 @@ public static Integer getMeasureAsInt(Orchestrator orch, String componentKey, St
   // The SonarQube alias "LTS" has been dropped. An alternative is "LATEST_RELEASE[6.7]".
   // The term "latest" refers to the highest version number, not the most recently published version.
   public static String replaceLtsVersion(String version) {
-    if (version != null && version.equals("LTS"))
-    {
+    if (version != null && version.equals("LTS")) {
       return "LATEST_RELEASE[7.9]";
     }
     return version;
   }
 
   public static void reset(Orchestrator orchestrator) {
+    // Only for local debugging:
+    if (orchestrator.getServer() == null) {
+      // Tests.ORCHESTRATOR is a jUnit rule that is automagically started in it's beforeAll() action for all tests.
+      // Running individual test doesn't execute the @ClassRule annotation on ORCHESTRATOR in Tests class.
+      orchestrator.start();
+    }
+
```

Comment:
As discussed, this will start orchestrator if needed for a local test run of an individual test.